README.md

```diff
@@ -61,3 +61,21 @@ Convert a JSON representation back into an immutable object
 ### `transit.withFilter(function) => transit`
 
 Create a modified version of the transit API that deeply applies the provided filter function to all immutable collections before serialising. Can be used to exclude entries.
+
+### `transit.withRecords([recordClasses])[.withFilter(function)] => transit`
```

Comment:
After further consideration and a bit of experimentation I haven't found a way I'm happy with to make this work while both preserving the existing semantics and not complicating the code internally. That doesn't mean it isn't possible, just that I don't see the way to do it. :smile: Given how infrequently I suspect someone will want to both set a filter and deal with Records, and given that the current solution works just fine, I'm inclined to leave it how it is right now.